Tests/HederaE2ETests/Topic/TopicMessageSubmit.swift

```diff
@@ -134,4 +134,53 @@ internal class TopicMessageSubmit: XCTestCase {
 
         _ = try XCTUnwrap(transaction.transactionId)
     }
+
+    internal func testSubmitMessage() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        let topic = try await Topic.create(testEnv)
+
+        addTeardownBlock {
+            try await topic.delete(testEnv)
+        }
+
+        let key = PrivateKey.generateEd25519()
+
+        let receipt = try await AccountCreateTransaction(key: .single(key.publicKey), initialBalance: 50)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let id = try XCTUnwrap(receipt.accountId)
+
+        let userClient = Client.forTestnet().setOperator(id, key)
```

Comment:
this is wrong (the test environment could be preview net or main net or some custom network that isn't testnet)